storey/flow.py

```diff
@@ -105,6 +116,26 @@ async def _do(self, event):
 
 
 class Event:
+    """The basic unit of data in storey. All steps receive and emit events.
+
+    :param body the event payload, or data
+    :param id: Event identifier.
+    :type id: string
+    :param key: Event key.
```

Comment:
Explain more about the logical difference and the usage difference between `id` and `key`.
Also, i'd put `key` first since it's more "interesting" in a flow

Reply:
It matches the order of the constructor params.

Reply:
From a user perspective, "Event key" and "Event identifier" are the same thing.. you have to explain what each of the does and how "we" use each of them.
regarding the order.. well in my opinion better order by significance, but as long as you explain both it's fine. BTW i would change the constructor params order to have key first from the same reasons...